sdk/core/azure-core/inc/azure/core/version.hpp

```diff
@@ -52,3 +52,4 @@ namespace Azure { namespace Core {
 #undef AZURE_CORE_VERSION_MINOR
 #undef AZURE_CORE_VERSION_PATCH
 #undef AZURE_CORE_VERSION_PRERELEASE
+
```

Comment:
We need to update the script to avoid adding this new line (if it keeps adding them). If it is just a one time thing, its fine.

cc @weshaggard

Reply:
Looks like we are missing "-NoNewLine" option to the Set-Content line at https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/scripts/Update-PkgVersion.ps1#L60.